### ISSUE TITLE
bonemenu never show up

### ISSUE THREAD

**Issue by elly500** [yowchap/BoneLib#37]
Include the following info or it won't be looked into.
Store (Steam, Oculus): 
Headset (Index, Quest 2, etc):
A detailed description of the issue: 
Steps to recreate the issue: 

Also upload your log file from **BONELAB\MelonLoader\Latest.log** even if there are no errors in it.

**Comment by elly500** [comment]
i am a queest and i have bonelab on quest and im using lemon loader and it not working

**Comment by Awesomcoin18** [comment]
Please fix i understand that moders work hard on making other peoples games more exciting so please

**Comment by DylanZutner** [comment]
If your not on the beta patch go to the version 2.2.0 instead of the latest

**Comment by Awesomcoin18** [comment]
Yes I tried that but the auto update changes it and because the file includes the term user data my pcs anti virus panicked

Sent from my T-Mobile 4G LTE Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by DylanZutner** [comment]
delete the auto update until they fix it
